1. Basic Web Development/index.html

```diff
@@ -23,63 +22,122 @@
         <div class="square8"></div>
         <div class="square9"></div>
     </div>
-    <div class="preferences hidden">
-        <div class="bar"></div>
-        <div class="content">
-            <h1>Select your preferences:</h1>
-            <div class="section">
-                <h2>Presets:</h2>
-                <form>
-                    <div class="grid-element">
-                        <label for="all">All</label>
-                        <input id="all" name="preset" type="radio" value="All" />
+    <div id="site-content" class="no-display">
+        <div class="begin-arrow">
+            <p>Begin Editing</p>
+            <p> |</p>
+            <p> |</p>
+            <p>\/</p>
+            <a class="to-begin no-decor" href="#create">Edit Pop-up</a>
+        </div>
+        <div class="editor no-display">
+            <div id="creation-page" class="">
+                <a class="to-edit no-decor" href="#edit">To edit ></a>
```

Comment:
The logic you used for edit is a bit weird and not so great UX. You have to pay attention to the list of permissions to see that buttons appeared in it. At least inform the user that he has to select one of the existing permissions listed bellow in order to edit them.